docs/rsdl/rapid-pro-rsdl-abnf.md

```diff
@@ -111,25 +120,27 @@ propertyName     = identifier / DQUOTE 1*CHAR DQUOTE
 ### Core Syntax Elements
 
 ```ABNF
-qualifiedName   = identifier *( "." identifier )
+qualifiedIdentifier = identifier *( "." identifier )
 
-identifier      = identInitial *identSubsequent
+identifier          = identInitial *identSubsequent
 
-identInitial    = ALPHA / "_" ; Note: actually all Unicode letters
+identInitial        = ALPHA / "_" ; Note: actually all Unicode letters
 
-identSubsequent = identInitial / DIGIT
+identSubsequent     = identInitial / DIGIT
 
-number          = DIGIT *DIGIT ["." *DIGIT ]
+number              = DIGIT *DIGIT ["." *DIGIT ]
 
-ALPHA  = %x41-5A / %x61-7A
+integer             = DIGIT *DIGIT
```

Comment:
"integer" should allow negative values, and for its use in `Scale` this is actually needed.